Google.Api.Generator/Utils/Typ.cs

```diff
@@ -56,6 +58,17 @@ private sealed class FromNested : Typ
             public override string FullName => $"{DeclaringTyp.FullName}+{Name}";
         }
 
+        private sealed class FromGeneric : Typ
+        {
+            public FromGeneric(Typ def, IEnumerable<Typ> typeArgs) => (_def, GenericArgTyps) = (def, typeArgs);
```

Comment:
Just to check, FromGeneric represents a constructed type, and def is the unbound generic type? It might be worth making a note of that somewhere.

Reply:
XmlDoc added.